### DISCUSSION TITLE
Moving while shooting bow or hookshot?

### DISCUSSION THREAD

**Discussion by qrstuvv** [leoetlino/project-restoration#225] [General]
I don't know, maybe since there are two sticks available now (one for camera and one for movement), i think it'd be pretty groovy if right stick + gyro = shooting adjustments and left stick = moving. and yes, i know you can do it while targeting but thats different i think it would be nice if i could keep it in first person also its way harder to aim and you cant move forward and back. (i also trip myself up sometimes because i try to use right stick instead of left to aim because i normally use gyro.) botw had it, why couldnt mm3d, right?

**Comment by leoetlino** [comment]
Yeah that sounds cool. I probably won't be working on this myself (as I have other projects to maintain and I've already spent a lot of time on MM3D) but I'll gladly accept a patch that implements this.
